afutests/memcpy/ocxl_memcpy.c

```diff
@@ -33,6 +33,7 @@
 #define LOG_INF(pid, fmt, x...) printf("Process %d: " fmt, pid, ##x)
 
 #define AFU_NAME "IBM,MEMCPY3"
+#define AFU_MAX_PROCESSES 512
```

Comment:
Is this a documented maximum, or should we query the AFU for it's maximum?

Reply:
This is the documented maximum.

Reply:
Ok, great, thanks for confirming